modules/recruiting/storage/output.tf

```diff
@@ -0,0 +1,39 @@
+output "resource_group_name" {
+  value = azurerm_resource_group.tikjob_rg.name
+}
+
+output "resource_group_location" {
+  value = azurerm_resource_group.tikjob_rg.location
+}
+
+/* MySQL */
+output "mysql_fqdn" {
+  value = azurerm_mysql_server.tikjob_mysql.fqdn
+}
+
+output "mysql_connection_user" {
```

Comment:
```suggestion
output "mysql_username" {
```

Reply:
Done in https://github.com/Tietokilta/tietokilta.fi-infra/pull/12/commits/51fe870d36f230e6b80a6c19f46f3539ee195c39